userservice/src/main/java/com/infodation/userservice/repositories/UserRepository.java

```diff
@@ -14,11 +14,11 @@
 
 @Repository
 public interface UserRepository extends JpaRepository<User, Long> {
-    @Query("SELECT u FROM users u " +
+    @Query("SELECT u FROM User u " +
             "WHERE :name = '' OR u.firstName LIKE %:name% OR u.lastName LIKE %:name%")
     Page<User> findByName(@Param("name") String name, Pageable pageable);
 
-    @Query("SELECT u.userId FROM users u")
+    @Query(value = "SELECT u.userId FROM User u", nativeQuery = true)
```

Comment:
I'm unsure why you changed the `users` (table name) to `User`. Is there any feasible reason?

Reply:
Because I have made some changes to the User model, the name of the entity is now 'User' instead of 'users'
![image](https://github.com/user-attachments/assets/1b0bc84c-c1cb-4e22-9782-69033fb65122)